templates/article-cover.php

```diff
@@ -12,14 +12,15 @@
  *
  * @see https://developers.facebook.com/docs/instant-articles/guides/articlecreate#specify-cover
  */
+
 ?>
 <header>
 
 	<?php
 
 	// Post featured image as FB IA cover image.
-	if ( $thumb_id = get_post_thumbnail_id() ) {
-		Simple_FB_Instant_Articles::instance()->render_image_markup( $thumb_id );
+	if ( has_post_thumbnail() ) {
+		Simple_FB_Instant_Articles::instance()->render_image_markup( get_post_thumbnail_id() );
```

Comment:
`has_post_thumbnail()` internally calls `get_post_thumbnail_id()` and casts it to `bool`. So I was just saving the need to ultimately call the same function twice.